pom.xml

```diff
@@ -912,10 +912,10 @@
 		<opencsv.version>1.8</opencsv.version>
 		<orbit.version.h2.database.engine>1.2.140.wso2v3</orbit.version.h2.database.engine>
 		<orbit.version.spark>1.1.0.wso2v1</orbit.version.spark>
-		<orbit.version.json>3.0.0.wso2v1</orbit.version.json>
+		<orbit.version.json>2.0.0.wso2v1</orbit.version.json>
 		<orbit.version.netty-all>4.0.23.wso2v1</orbit.version.netty-all>
-		<orbit.version.axis2>1.6.1.wso2v11</orbit.version.axis2>
-		<orbit.version.axiom>1.2.11.wso2v5</orbit.version.axiom>
+		<orbit.version.axis2>1.6.1.wso2v12</orbit.version.axis2>
+		<orbit.version.axiom>1.2.11.wso2v6</orbit.version.axiom>
 		<orbit.version.htrace>3.0.4.wso2v1</orbit.version.htrace>
 		<orbit.version.jettison>1.3.4.wso2v1</orbit.version.jettison>
 		<axis2.wso2.version>${orbit.version.axis2}</axis2.wso2.version>
```

Comment:
I assume you ran the build with all tests. If not please run mvn clean install since here we're moving to Carbon kernel 4.4.